src/Components/Calendar/Calendar.tsx

```diff
@@ -0,0 +1,277 @@
+import {
+  CalendarEvent,
+  PopupEvent,
+  defaultEventLength as defaultEveLength,
+  defaultView,
+} from "./calendarutils/calendarutils";
+
+const localizer = momentLocalizer(moment);
+
+type Keys = keyof typeof Views;
+
+type Props = {
+  events: CalendarEvent[];
+  backgroundEvents: { id: number; title: string; start: Date; end: Date };
+  availability?: { start: string; end: string };
+  defaultEventLength?: number;
+  athletes: [{ option: string; value: string }];
+};
+
+export const Calendar: Story<Props> = ({
+  events,
+  backgroundEvents,
+  availability,
+  defaultEventLength = defaultEveLength,
+  athletes,
+}: Props) => {
+  const [date, setDate] = useState<Date>(moment(new Date()).toDate());
+  const [view, setView] = useState<(typeof Views)[Keys]>(Views.MONTH);
+  const [toggleValue, setToggleValue] = useState(defaultView);
+  const [myEvents, setEvents] = useState<CalendarEvent[]>(events);
+  const [selected, setSelected] = useState();
+  const [eventPopup, setEventPopup] = useState<PopupEvent>();
+  const [show, setShow] = useState(false);
+
+  const currentMonth = date.toLocaleString("default", { month: "long" });
+
+  const toolbarTitle = new Map();
+  toolbarTitle.set("month", `${currentMonth} ${date.getFullYear()}`);
+  toolbarTitle.set("week", `${currentMonth} ${date.getFullYear()}`);
+  toolbarTitle.set(
+    "day",
+    `${date.getDate()}-${date.getMonth() + 1}-${date.getFullYear()}`
+  );
+
+  class CustomToolbar extends Toolbar {
+    handleNavigation = (value: string) => {
+      switch (value) {
+        case "Previous":
+          this.navigate("PREV");
+          break;
+        case "Today":
+          this.navigate("TODAY");
+          break;
+        case "Next":
+          this.navigate("NEXT");
+          break;
+      }
+    };
+
+    render() {
+      return (
+        <div className="rbc-toolbar">
+          <div className={styles.toolbarNav}>
+            <Toggle
+              name="NavButtons"
+              values={["Previous", "Today", "Next"]}
+              defaultValue=""
+              setChecked={this.handleNavigation}
+            />
+          </div>
+          <p className="rbc-toolbar-label">{this.props.label}</p>
+          <Toggle
+            name="viewSelect"
+            values={["Month", "Week", "Day"]}
+            setChecked={changeView}
+            defaultValue={toggleValue}
+          />
+        </div>
+      );
+    }
+
+    navigate = (action) => {
+      this.props.onNavigate(action);
+    };
+  }
+
+  const onView = useCallback(
+    (newView) => {
+      setView(newView);
+      setToggleValue("Day");
+    },
+    [setView]
+  );
+
+  //Week Label
+  const dayRangeHeaderFormat = (
+    { start, end },
+    culture,
+    localizer: momentLocalizer
+  ) => {
+    const sameMonth = localizer.eq(start, end, "month");
+    const sameYear = localizer.eq(start, end, "year");
+    const formatter = `DD ${sameMonth ? "" : "MMM"} ${sameYear ? "" : "YYYY"}`;
```

Comment:
Avoid using multiple ternaries in the same place, especially when the default value is nothing - you can just optionally render it